quickwit-backward-compat/README.md

```diff
@@ -3,7 +3,50 @@
 This is just a project used to test backward compatibility of Quickwit.
 Right now, only SplitMetadata is tested.
 
-The build.rs script autocreates json sample files when something changes in the format.
-A unit test then checks that all of these sample files deserialize to the right
-`SplitMetadataAndFooterOffsets` object.
+We want to be able to read all past version of SplitMetadata, but only write the most recent format.
+The tests consists in pairs of json file.
+XXXX.json and XXXX.expected.json
```

Comment:
wasn't it like?

vX-[sha].json

Reply:
The sha1 is part of the XXXX. You are not required to include any sha1 in the name by the way. It is possible to add manually crafter use cases.